lib/auth/types/AuthType.js

```diff
@@ -201,7 +210,7 @@ export default class AuthType {
                 try {
                     credentials = await this.server.auth.test('security_access_control_cookie', request);
                     return h.authenticated({credentials})
-                 } catch(error) {
+                } catch(error) {
```

Comment:
would you help to revert the code formation change, it will help understand what's the actual change.